### PR TITLE
fixing courses and features grid size and adding links

### PR DESCRIPTION
**Pull request by Stiffpixels**
- changed the grid by giving it a minimum height so every grid item is even
- added links to each card title

current:
image

after my changes:
image

### CODE REVIEW

File: src/components/HomePage/home.module.css [hunk 1]
```diff
@@ -156,6 +157,11 @@
   margin-top: 5px;
 }
 
+.feature_links,
+.course_links{
+  color: black ;
```
Comment (Ajay-Dhangar) [inline src/components/HomePage/home.module.css:162]: removed color

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
can you remove only color of link, you assign in css `color: black` and all is fine

File: src/components/HomePage/home.module.css (hunk 1 above)
Comment (Stiffpixels) [reply src/components/HomePage/home.module.css:162]: ok sorry forgot to remove that

**Comment by Stiffpixels** [comment]
also please assign a label to this pr as well like in the issue #78

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Comment by Stiffpixels** [comment]
> Done

Can you add the level as well

**Pull request merged**